src/main/java/lotto/LottoManager.java

```diff
@@ -34,10 +40,30 @@ public static void printResult(LottoGroup lottoGroup, WinningLotto winningLotto)
     private static LottoGroup createLottoGroup(Money money) {
         try {
             LottoSeller lottoSeller = new LottoSeller();
+            int count = InputView.getManualLottoCount();
+            if (count != 0) {
+                List<Lotto> lottos = createManualLotto(InputView.getManualNumbers(count));
+                return lottoSeller.sellManualLotto(money, count, lottos);
+            }
             return lottoSeller.sellLotto(money);
         } catch (LottoException e) {
             OutputView.printMessage(e.getMessage());
             return createLottoGroup(InputView.getMoney());
+        } catch (NumberFormatException e) {
+            OutputView.printMessage("숫자를 입력해주세요.");
+            return createLottoGroup(InputView.getMoney());
+        }
+    }
+
+    private static List<Lotto> createManualLotto(List<String> numbers) {
+        List<Lotto> lottos = new ArrayList<>();
+        for (String number : numbers) {
+            lottos.add(LottoGenerator.generate(Arrays
+                    .stream(number.split(","))
+                    .mapToInt(Integer::parseInt)
```

Comment:
질문주신 세번째 부분인거같아요!
```suggestion
                    .map(Integer::parseInt)
```
그냥 이렇게 해도되지않을까요? 질문주신,

> 제가 boxed를 대략 찾아보니 int -> Integer로 포장하는 메소드라고 하는데요.
> boxed를 쓰는 것이 다시 포장하는 것이 꺼려지고 또한 번거롭다고 생각하는데 boxed 이외에 ....

boxed는 IntStream을 다시 일반 Stream으로 변환할 때 사용합니다. 주로 계산할때 언박싱 비용을 회피하기 위해서 사용하는데요, (Integer의 sum을보시면 원시형으로 계산하기 때문에 언박싱을 해주어야합니다) 
지금은 계산도없고.. Integer를 이용하여 무언가 조작이 있는게 아닌걸로보여 굳이 저 형식을 사용할필요는 없다는 생각입니다.